index.html

```diff
@@ -214,5 +212,25 @@ <h2>Languages <i class="fas fa-angle-down"></i></h2>
       <li><h3>Skills <i class="fas fa-angle-right"></i></h3></li>
 
 </ul>
+
+
+<footer class="footer">
+    <div class="footer-wrap">
+      <h3>Contact me</h3>
+    <p>If you have an application you <br> are interested in developing, a <br> feature that you need built or a <br> project that needs coding. I’d <br>love to help with it </p>
+    
+    <form class="contact_form" action="/contact_form" method="post">
+
+               <input type="text" id="name" Placeholder="Odette Gleen" maxlength="30" required>
+               <input type="text" id="email" placeholder="nkargleeny@gmail.com" required>
+               <textarea id="msg" placeholder="Write your message here" maxlength="500" required></textarea>
+          
+
+        <button type="button" class="button"> Get It Touch</button>
```

Comment:
- [ ] Kindly add button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.
<img width="459" alt="Screenshot 2022-05-04 at 7 04 50 PM" src="https://user-images.githubusercontent.com/70269507/166723376-ae54c022-b783-47b5-bc48-05972220c4ec.png">